MVP_Clicker/Assets/Project/Scripts/Game/Areas/Monster/Config/MonsterConfig.cs

```diff
@@ -10,13 +10,15 @@ public class MonsterConfig : ScriptableObject, IMonsterConfig
         [SerializeField] private int _startRewardForKilling;
         [SerializeField] private string _typeOfRewardForKilling;
         [SerializeField] private string _resourceDamagingMonster;
+        [SerializeField] private string _resourceDamagingMonsterEverySecond;
         [SerializeField] private Sprite _startMonsterImage;
         [SerializeField] private List<Sprite> _monsterImages;
 
         public int StartFullHp => _startFullHp;
         public int StartRewardForKilling => _startRewardForKilling;
         public Sprite StartMonsterImage => _startMonsterImage;
         public string ResourceDamagingMonster => _resourceDamagingMonster;
+        public string ResourceDamagingMonsterEverySecond => _resourceDamagingMonsterEverySecond;
```

Comment:
it's little not clear enoght that is this field for. May be `_damagingMonsterEverySecondResourceId `more clear?